fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+swagger: '2.0'
+  
+info:
+  version: 1.0.0 
+  title: Draft Orders API
+  x-logo:
+    url: "https://andrewjtech123.github.io/imagehost/shopify_logo_black.png"
+    backgroundColor:
+  
+  description: >
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the order is set to `completed` status. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
+      
+      * Create a draft order that calculates taxes and totals but accept payment from the customer outside of Shopify.
```

Comment:
* noticing that the bulleted lists receive bold formatting -- would recommend changing the template to use regular weight throughout, reserving bold for emphasis when it's required 
* consider "totals but accept" -> "totals**,** but accept" 
* "outside of Shopify" -> "outside Shopify"